src/forms/TimeInput.jsx

```diff
@@ -27,7 +27,7 @@ export const getTimeParts = (time) => {
 	};
 };
 const formatHours = (hours, meridian) => {
-	const newHours = meridian ? (parseInt(hours) + (12 * (meridian === 'PM'))) : hours % 24 ;
+	const newHours = meridian ? (parseInt(hours % 12) + (12 * (meridian === 'PM'))) : hours % 24 ;
```

Comment:
also not a blocker, but a little too much going on to pick this up at a glance

Reply:
always best practice to include the radix with parseInt()

i.e. `parseInt(hours % 12, 10)`